Sources/ComposableArchitecture/Store.swift

```diff
@@ -195,20 +195,22 @@ public final class Store<State, Action> {
 
 /// A producer of store state.
 @dynamicMemberLookup
-public struct StoreProducer<State>: SignalProducerConvertible {
-  public let producer: Effect<State, Never>
+public struct Produced<Value>: SignalProducerConvertible {
+  public let producer: Effect<Value, Never>
 
-  init(_ upstream: Effect<State, Never>) {
+  init(by upstream: Effect<Value, Never>) {
```

Comment:
Why did you add `by` here?

Reply:
Ok, so the init looks nicer 👍  Maybe use `with` instead of `by` ?

Reply:
That is an internal change and will not affect framework users, so I decided that the call site will look nicer.
I like both `by` and `with`, it's up to you :)
- `Produced(by: producer)`
- `Produced(with: producer)`

Reply:
Ok, I'm fine with `by` 😄